tests/utils_tests/test_crypto.py

```diff
@@ -2,22 +2,38 @@
 import unittest
 
 from django.test import SimpleTestCase
+from django.test.utils import ignore_warnings
 from django.utils.crypto import (
     InvalidAlgorithm,
     constant_time_compare,
     pbkdf2,
     salted_hmac,
 )
+from django.utils.deprecation import RemovedInDjango61Warning
 
 
 class TestUtilsCryptoMisc(SimpleTestCase):
+    # RemovedInDjango61Warning: Remove this test.
+    @ignore_warnings(category=RemovedInDjango61Warning)
     def test_constant_time_compare(self):
         # It's hard to test for constant time, just test the result.
         self.assertTrue(constant_time_compare(b"spam", b"spam"))
         self.assertFalse(constant_time_compare(b"spam", b"eggs"))
         self.assertTrue(constant_time_compare("spam", "spam"))
         self.assertFalse(constant_time_compare("spam", "eggs"))
 
+    def test_constant_time_compare_deprecated(self):
+        msg = (
+            "constant_time_compare() is deprecated. "
+            "Use hmac.compare_digest() instead."
+        )
+        with self.assertWarnsMessage(RemovedInDjango61Warning, msg) as ctx:
+            self.assertTrue(constant_time_compare(b"spam", b"spam"))
+            self.assertFalse(constant_time_compare(b"spam", b"eggs"))
+            self.assertTrue(constant_time_compare("spam", "spam"))
+            self.assertFalse(constant_time_compare("spam", "eggs"))
```

Comment:
```suggestion
            constant_time_compare(b"spam", b"spam")
```
This doesn't need to repeat the test assertions of the previous test